library/Icingadb/Widget/Detail/UsergroupDetail.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/* Icinga DB Web | (c) 2021 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Icingadb\Widget\Detail;
+
+use Icinga\Module\Icingadb\Common\Auth;
+use Icinga\Module\Icingadb\Common\Links;
+use Icinga\Module\Icingadb\Model\Usergroup;
+use Icinga\Module\Icingadb\Widget\EmptyState;
+use Icinga\Module\Icingadb\Widget\ItemList\UserList;
+use Icinga\Module\Icingadb\Widget\ShowMore;
+use ipl\Html\BaseHtmlElement;
+use ipl\Html\HtmlElement;
+use ipl\Html\Text;
+
+class UsergroupDetail extends BaseHtmlElement
+{
+    use Auth;
+
+    /** @var Usergroup The given user group */
+    protected $usergroup;
+
+    protected $defaultAttributes = ['class' => 'usergroup-detail'];
+
+    protected $tag = 'div';
+
+    public function __construct($usergroup)
```

Comment:
The docblock for the `$usergroup` property hints at `Usergroup`, so please also require the `$usergroup` param to be an instance of that.